frappe/public/js/frappe/views/kanban/kanban_board.js

```diff
@@ -590,8 +590,7 @@ frappe.provide("frappe.views");
 
 		function bind_edit_card() {
 			self.$card.find('.kanban-card.content').on('click', function() {
-				frappe.set_route('Form', card.doctype, card.name);
-				// setup_edit_card();
+				window.open('#Form/'+card.doctype+'/'+card.name);
```

Comment:
This works, but just change for readability

```suggestion
				window.open(frappe.utils.get_form_link(card.doctype, card.name));
```

Reply:
Updated in latest commit.